### DISCUSSION TITLE
Add Qwik version of components

### DISCUSSION THREAD

**Discussion by SimplySayHi** [tailwindlabs/headlessui#2537] [Ideas]
Hi, is there a plan to create HeadlessUI components for Qwik framework?

**Comment by IRediTOTO** [comment]
i want this too :D

**Comment by pleopardi** [comment]
Did you try to use [Qwik React](https://qwik.builder.io/docs/integrations/react) to make it work? I'm interested too and will probably give it a try in a few days

**Reply by SimplySayHi** [reply]
Nope, btw Qwik documentation says Qwik React is just a migration strategy with a lot of limitations, and abuse can cause performance issues

**Reply by pleopardi** [reply]
Yes, you lose resumability on those components, a native implementation would definitely be better.
